### PR TITLE
Add governance doc

### PR DESCRIPTION
**Pull request by ErikJiang**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Add kubean open source community governance description document.

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+4. share experiences with the Kubean project and actively promote it.
+5. submit suggestions/criticisms.
+
+Prominent contributors can expect to be included in the Kubean project's [Maintainers List](https://github.com/kubean-io/kubean/blob/main/MAINTAINERS.md).
+All contributors who have submitted PRs to Kubean will be permanently recorded in the [Contributors List](https://github.com/kubean-io/kubean#thumbsup-contributors).
+
+
+### Core Technical Committee
+
+The Kubean project is governed and directed by a Core Technical Committee, which is responsible for the development of the Kubean project.
```
Comment (windsonsea) [inline GOVERNANCE.md:25]:
```suggestion
The Kubean project is governed and directed by a Core Technical Committee (CTC), which is responsible for the development of the Kubean project.
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+The CTC has final authority over this project including:
```
Comment (windsonsea) [inline GOVERNANCE.md:27]:
```suggestion
The CTC has final authority over this project, which includes:
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
```
Comment (windsonsea) [inline GOVERNANCE.md:34]:
```suggestion
* Setting the technical direction
* Establishing project governance and processes (including this policy)
* Managing the contribution policy
* Overseeing the GitHub repository hosting
* Enforcing conduct guidelines
* Maintaining the list of additional Collaborators
```

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+The Core Technical Committee consists of Reviewers & Approvers, see **[OWNERS](https://github.com/kubean-io/kubean/blob/main/OWNERS "OWNERS")** for a list.
+
+### Communication
+1. we recommend [Github Issue](https://github.com/kubean-io/kubean/issues "Github Issue") for tracking bugs and managing requirements.
+2. we recommend [Github Milestone](https://github.com/kubean-io/kubean/issues "GitHub Milestone") for managing project progress and planning;
```
Comment (windsonsea) [inline GOVERNANCE.md:40]:
```suggestion
1. We recommend using [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues") for tracking bugs and managing requirements;
2. We recommend using [Github Milestone](https://github.com/kubean-io/kubean/issues "GitHub Milestone") for managing project progress and planning.
```

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+### Documentation
+1. We recommend using [Github Pages](https://kubean-io.github.io/kubean "Github Pages") to deposit the documentation;
+2. we recommend using [Github Wiki](https://github.com/kubean-io/kubean/wiki "Github Wiki") to deposit FAQs; `TODO`.
```
Comment (windsonsea) [inline GOVERNANCE.md:44]:
```suggestion
2. We recommend using [Github Wiki](https://github.com/kubean-io/kubean/wiki "Github Wiki") to deposit FAQs; `TODO`.
```

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+# Kubean Project Governance
+
+### User
+
+Kubean community users, feel free to comment your usage information in **[Who is using Kubean](https://github.com/kubean-io/kubean/issues/910 "Who is Using Kubean")**.
+
+The community welcomes any individuals, companies, and organizations to use, develop, feedback bugs, submit feature requests, and help each other. We recommend using [Github Issue](https://github.com/kubean-io/kubean/issues "Github Issue") to track bugs and manage requests.
```
Comment (windsonsea) [inline GOVERNANCE.md:7]:
```suggestion
The community welcomes any individuals, companies, and organizations to use, contribute to, report bugs, submit feature requests, and help each other.
We recommend using [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues") to track bugs and manage requests.
```

File: GOVERNANCE.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+### Contributor
+
+Every user is welcome to participate and contribute to the Kubean open-source community in a number of ways, including but not limited to the following:
+
+1. participate in the community by actively engaging in discussions in [Github Issue](https://github.com/kubean-io/kubean/issues "Github Issue").
```
Comment (windsonsea) [inline GOVERNANCE.md:13]:
```suggestion
1. engage in lively discussions in [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues").
```

**Review by windsonsea — COMMENTED** [review]
Some test suggestions

File: GOVERNANCE.md (hunk 7 above)
Comment (ErikJiang) [reply GOVERNANCE.md]: 😁Thanks, @windsonsea, I'm done.

File: GOVERNANCE.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+# Kubean Project Governance
+
+### User
+
+Kubean community users, feel free to comment your usage information in **[Who is using Kubean](https://github.com/kubean-io/kubean/issues/910 "Who is Using Kubean")**.
```
Comment (Michelle951) [inline GOVERNANCE.md:5]:
```suggestion
Kubean community users, feel free to feedback your use of Kubean in **[Who is using Kubean](https://github.com/kubean-io/kubean/issues/910 "Who is Using Kubean")**.
```

File: GOVERNANCE.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+We recommend using [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues") to track bugs and manage requests.
+
+### Contributor
+
+Every user is welcome to participate and contribute to the Kubean open-source community in a number of ways, including but not limited to the following:
+
+1. engage in lively discussions in [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues").
+2. submit code patches.
+3. translate, revise, add to, and improve the [documentation](https://kubean-io.github.io/kubean/ "Documentation");
+4. share experiences with the Kubean project and actively promote it.
```
Comment (Michelle951) [inline GOVERNANCE.md:18]:
```suggestion
4. Share experiences with the Kubean project and actively promote it.
```

File: GOVERNANCE.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+5. submit suggestions/criticisms.
```
Comment (Michelle951) [inline GOVERNANCE.md:19]:
```suggestion
5. Submit suggestions/criticisms.
```

File: GOVERNANCE.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+Prominent contributors can expect to be included in the Kubean project's [Maintainers List](https://github.com/kubean-io/kubean/blob/main/MAINTAINERS.md).
```
Comment (Michelle951) [inline GOVERNANCE.md:21]:
```suggestion
Prominent contributors can be included in the Kubean project's [Maintainers List](https://github.com/kubean-io/kubean/blob/main/MAINTAINERS.md).
```

Comment (ErikJiang) [reply GOVERNANCE.md]: hi @Michelle951, I'm done.😀

Comment (Michelle951) [reply GOVERNANCE.md]: /lgtm

**Pull request merged**